src/components/FormAddRoom.jsx

```diff
@@ -0,0 +1,128 @@
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    createRoom({
+      name,
+      description,
+      wifi,
+      tv,
+      room_service: roomService,
+      beds,
+      image_url: imageUrl,
+      reserved,
+    });
+  };
+
+  return (
+    <div className="flex justify-center w-[1200px] h-screen items-center">
+      <form
+        className="bg-neutral-50 flex flex-col justify-center items-center w-[450px] h-[650px] text-black"
+        onSubmit={handleSubmit}
+      >
+        <label htmlFor={name}>
+          <span>Name</span>
+          <br />
+          <input
+            type="text"
+            value={name}
+            onChange={(e) => setName(e.target.value)}
+            className="w-[300px]"
+          />
+        </label>
+        <br />
+        <label htmlFor={description}>
+          <span>Description</span>
+          <br />
+          <textarea
+            value={description}
+            onChange={(e) => setDescription(e.target.value)}
+            className="w-[300px]"
+          />
+        </label>
+        <br />
+        <label htmlFor={wifi}>
+          <span>Wi-Fi</span>
+          <br />
+          <input
+            type="text"
+            value={wifi}
+            onChange={(e) => setWifi(e.target.value)}
+            className="w-[300px]"
+          />
+        </label>
+        <br />
+        <label htmlFor={tv}>
+          <span>TV</span>
+          <br />
+          <input
+            type="text"
+            value={tv}
+            onChange={(e) => setTv(e.target.value)}
+            className="w-[300px]"
+          />
+        </label>
+        <br />
+        <label htmlFor={roomService}>
+          <span>Room Service</span>
+          <br />
+          <input
+            type="text"
+            value={roomService}
+            onChange={(e) => setRoomService(e.target.value)}
+            className="w-[300px]"
+          />
+        </label>
+        <br />
+        <label htmlFor={beds}>
+          <span>Beds</span>
+          <br />
+          <input
+            type="number"
+            value={beds}
+            onChange={(e) => setBeds(e.target.value)}
+            className="w-[300px]"
+          />
+        </label>
+        <br />
+        <label htmlFor={imageUrl}>
+          <span>Image URL</span>
+          <br />
+          <input
+            type="text"
+            value={imageUrl}
+            onChange={(e) => setImageUrl(e.target.value)}
+            className="w-[300px]"
+          />
+        </label>
+        <br />
+        <label htmlFor={reserved} className="flex flex-row items-center gap-2">
+          <input
+            type="checkbox"
+            checked={reserved}
+            onChange={(e) => setReserved(e.target.checked)}
+          />
+          <span>Reserved</span>
+        </label>
+        <br />
+        <button type="submit" className="bg-lime-500 w-[180px] h-[35px]">Create Room</button>
+      </form>
```

Comment:
[OPTIONAL] I think having a consistent design in buttons would be a good idea for this project. Using the same design as the Login button would help your user's experience and show that you have great attention to detail 💯